codes/codes_test.go

```diff
@@ -91,3 +91,12 @@ func (s) TestUnmarshalJSON_MarshalUnmarshal(t *testing.T) {
 		}
 	}
 }
+
+func (s) TestUnmarshalJSON_InvalidIntegerCode(t *testing.T) {
+	var got Code
+	wantErr := "invalid code: 200" // for integer invalid code, expect integer value in error message
+	err := got.UnmarshalJSON([]byte("200"))
+	if err.Error() != wantErr {
+		t.Errorf("got.UnmarshalJSON(200) != %s; got=%s", wantErr, err.Error())
+	}
```

Comment:
Consider rewriting as:

```golang
	if err := got.UnmarshalJSON([]byte("200")); err != wantErr {
		t.Errorf("UnmarshalJSON(200) = %v, wantErr: %v", err, wantErr)
	}
```

The important difference above is `got before want`. See https://google.github.io/styleguide/go/decisions#got-before-want

Reply:
Done